index.html

```diff
@@ -11,23 +11,55 @@
     <link href="https://fonts.googleapis.com/css2?family=Hi+Melody&display=swap" rel="stylesheet" />
   </head>
   <body>
-    <section class="first-page">
-      <section class="main">
-        <h1 class="main-title">All awesome books</h1>
-        <ul class="list-books" id="booksList"></ul>
-      </section>
+    <div class="home">
+      <nav class="navbar">
+        <p class="navbar-title">Awesome Books</p>
+        <ul class="navbar-list">
+          <li><a class="link" href="#" id="list">List</a></li>
+          <li class="column"></li>
+          <li><a class="link" href="#" id="addNew">Add new</a></li>
+          <li class="column"></li>
+          <li><a class="link" href="#" id="contact">Contact</a></li>
+        </ul>
```

Comment:
- [ ] To enhance the user experience of this design kindly ensure that an active menu link is highlighted(color change or change in opacity). In other words, the current active page should have the corresponding menu item highlighted as can be seen in the provided [wireframe design](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m4_full_website.md), as we really want our designs to as much as possible mimic the provided wireframe template..

**Your design menu:**
![image](https://user-images.githubusercontent.com/12745474/130077336-8767c1ec-614e-44d3-8cc5-e7778e8d23d9.png)


**The wireframe menu design:**
![image](https://user-images.githubusercontent.com/12745474/130077612-f3d5a365-2c1d-4108-9636-6dfa6e770bb3.png)